src/constants.ts

```diff
@@ -0,0 +1 @@
+export const timeoutDurationMS = 5_000;
```

Comment:
Consider documenting here that this timeout is for each attempt, how many attempts there will be, and what feedback the user gets

Reply:
Done